client-registry-jempi/docker-compose.api.yml

```diff
@@ -30,6 +30,13 @@ services:
         - traefik.http.routers.jempi-api.service=jempi-api
         - traefik.http.services.jempi-api.loadbalancer.server.port=50000
         - traefik.http.routers.jempi-api.rule=Host(`${JEMPI_API_TRAEFIK_SUBDOMAIN}.${DOMAIN_NAME_HOST_TRAEFIK}`)
+        - traefik.http.routers.jempi-api.entrypoints=websecure
+        - traefik.http.routers.jempi-api.tls=true
```

Comment:
With the previous reverse proxy (nginx) we allowed a secure mode and an insecure mode (via the INSECURE env var). I think this might force the use of HTTPS always. This might not always be what we want. I suggest that we support the same env vars with Traefik just like we do with nginx.